src/dev/flang/tools/docs/Util.java

```diff
@@ -78,11 +78,11 @@ static String commentOf(AbstractFeature af)
       }
     var line = af.pos().line() - 1;
     var commentLines = new ArrayList<String>();
-    while (true)
+    while (line > 0)
       {
         var pos = new SourcePosition(af.pos()._sourceFile, af.pos()._sourceFile.lineStartPos(line));
         var strline = Util.lineAt((LibraryFeature) af, pos);
-        if (line < 1 || !strline.matches("^\\s*#.*"))
+        if (!strline.matches("^\\s*#.*"))
           {
             break;
```

Comment:
I am very unhappy to see `break` and `return` in the middle of some nested code. `while (true)` is a clear code smell.
Try if this works or do something similar: 
```
    var sf = af.pos()._sourceFile;
    while (line > 0 && sf.line(line).matches("^\\s*#.*"))
      {
        commentLines.add(sf.line(line));
        line = line - 1;
      }
```
I think `Util.lineAt` is complete overkill, or is there some reason for this to read to whole file every time?